hotel/src/test_dcrypto.rs

```diff
@@ -0,0 +1,38 @@
+//! Test DCRYPTO hardware
+
+use kernel::returncode::ReturnCode;
+#[allow(unused_imports)]
```

Comment:
Generally avoid, but it can be OK in tests because they might use the abstraction in a limited (and not fully functional way), such that the standard import approach has unused parts. However, in this case, it can be removed. Good catch. :)